.github/workflows/lint.yml

```diff
@@ -0,0 +1,22 @@
+---
+name: lint
+
+on:
+  push:
+    branches:
+      - main
+    tags:
+  pull_request:
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v4
+        with:
+          python-version: '3.9'
+      - name: install tox
+        run: python -m pip install --upgrade tox virtualenv setuptools pip
+      - name: run linter
+        run: tox -ebashate
```

Comment:
shouldn't we run the examples ?

Reply:
Why wouldn't we ? If we have something that assures a PR does not break the examples, shouldn't we rely on it ?